genson/src/main/java/com/owlike/genson/reflect/VisibilityFilter.java

```diff
@@ -70,7 +70,14 @@ public VisibilityFilter(int... modifier) {
    * @return true if this member is visible according to this filter.
    */
   public final boolean isVisible(Member member) {
-    return isVisible(member.getModifiers());
+    Class<?> clazz = member.getDeclaringClass();
+    String className = clazz.getName();
+    if(className.startsWith("java.") || className.startsWith("javax.")){
+      return Modifier.isPublic(member.getModifiers());
```

Comment:
I think this is not enough. The current impl of isVisible does apply a filter on things like transient, static, volatile, etc. You probably still want to leverage the same filter logic. It's worth adding some tests too.

Reply:
I could probably reverse the steps.  Run the existing visibility check first, then the additional check for java/javax class afterwards

Reply:
Yep this sounds like a good option.